lib/src/modules/task.cc

```diff
@@ -108,6 +123,10 @@ Task::Task(const fs::path& exec_prefix,
 
     input_validator_.registerSchema(input_schema);
     results_validator_.registerSchema(output_schema);
+
+    client_.set_ca_cert(ca);
```

Comment:
I had to mess with how I build pxp-agent to get matching versions of OpenSSL and libcurl:
```
cmake -DCMAKE_PREFIX_PATH="/usr/local/opt/openssl@1.1;/usr/local/opt/curl" ..
```